.storybook/styles.css

```diff
@@ -0,0 +1,70 @@
+.dark {
```

Comment:
This css styles the error/loading/docs loading UIs in a dark theme that matches the rest of the storybook theme.